src/pages/Inquiry/pagination.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactComponent as ArrowLeft } from 'assets/svg/inquiry/arrow-left.svg';
+import { ReactComponent as ArrowRight } from 'assets/svg/inquiry/arrow-right.svg';
+import { ReactComponent as DoubleArrowLeft } from 'assets/svg/inquiry/double-arrow-left.svg';
+import { ReactComponent as DoubleArrowRight } from 'assets/svg/inquiry/double-arrow-right.svg';
+import styles from './pagination.module.scss';
+
+interface Props {
+  totalPage: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+  page: number;
+}
+
+export default function Pagination({ totalPage, setPage, page }: Props): JSX.Element {
+  const backTenPage = () => {
+    setPage((prev) => (prev > 10 ? prev - 10 : 1));
+  };
+  const backOnePage = () => {
+    setPage((prev) => (prev > 1 ? prev - 1 : 1));
+  };
+
+  const nextOnePage = () => {
+    setPage((prev) => (prev + 1 <= totalPage ? prev + 1 : totalPage));
+  };
+
+  const nextTenPage = () => {
+    setPage((prev) => (prev + 10 <= totalPage ? prev + 10 : totalPage));
+  };
```

Comment:
위 리뷰에서도 언급했듯, 하나의 코드로 줄일 수 있을 것 같아요.
한번 이해하고 적용해보세여
```suggestion
  const movePage = (offset: number) => {
    setPage((prev) => {
      if(prev + offset < 1) return 1;
      if(prev + offset > totalPage) return totalPage;
      return prev + offset;
    });
  };
```

Reply:
56a6998d96443b7bd426c42cef55e92045bdad30 
넵! 적용 했습니다!! 감사합니다!